chargebee/src/main/java/com/chargebee/android/billingservice/CBPurchase.kt

```diff
@@ -96,23 +102,24 @@ object CBPurchase {
         }
     }
 
+    /**
+     * This method will provide all the purchases associated with the current account based on the [inActivePurchases] flag set.
+     * And the associated purchases can be synced with Chargebee.
```

Comment:
"And the associated purchases _**will**_ be synced with Chargebee." ?

Reply:
Done